jenkins/scripts/integration_test.sh

```diff
@@ -37,6 +37,7 @@ CAPM3_VERSION="${CAPM3_VERSION:-v1beta1}"
 NUM_NODES="${NUM_NODES:-2}"
 TESTS_FOR="${TESTS_FOR:-integration_test}"
 BARE_METAL_LAB=false
+PROMETHEUS_MONITORING="${PROMETHEUS_MONITORING:-false}"
 
```

Comment:
don't we have its default value set in JJB?

Reply:
only for feature test. In our case airship dev tools integration test, it is not set. That`s why I was thinking we set false value if it is not set.

Reply:
yep makes sense.